deps/graph-parser/src/logseq/graph_parser/property.cljs

```diff
@@ -12,6 +13,31 @@
    (contains? #{"Property_Drawer" "Properties"}
               (first block))))
 
+(def markers
+  #{"now" "later" "todo" "doing" "done" "wait" "waiting"
+    "canceled" "cancelled" "started" "in-progress"})
+
+(def built-in-extended-properties (atom #{}))
+(defn register-built-in-properties
+  [props]
+  (reset! built-in-extended-properties (set/union @built-in-extended-properties props)))
+
+(defn editable-built-in-properties
+  "Properties used by logseq that user can edit"
+  []
+  #{:title :alias :tags :template :template-including-parent :public})
+
+(defn hidden-built-in-properties
+  "Properties used by logseq that user can't edit or see"
+  []
+  (set/union
+   #{:id :custom-id :background-color :background_color :heading :collapsed
+     :created-at :updated-at :last-modified-at :created_at :last_modified_at
+     :query-table :query-properties :query-sort-by :query-sort-desc :ls-type
+     :hl-type :hl-page :hl-stamp :icon :filters :file-path}
```

Comment:
`filters` here makes me think that we shouldn't show it when editing page properties. :)

Reply:
I added `:icon :filters :file-path` to the existing set because I thought they were already hidden and accidentally not included in this definition. If any of these three are currently editable, I'm happy to move them up to the editable fn

Reply:
I think :icon should be editable since people can't change page icon if not.

Reply:
I've removed the `icon::` property from list. `icon::` should be editable by users.